src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -302,7 +302,9 @@ public function request($method, $uri, array $parameters = [], array $files = []
             $server['HTTP_HOST'] = $this->extractHost($uri);
         }
 
-        $server['HTTPS'] = 'https' == parse_url($uri, PHP_URL_SCHEME);
```

Comment:
I think we should do the smallest possible change.
What about this?
`$server['HTTPS'] = (int) ('https' == parse_url($uri, PHP_URL_SCHEME));`

Reply:
Isn't it a BC break?

Reply:
@fabpot i just rechecked the whole thing. In fact, the "https" value always gets redefined in the client object, no matter which value `$server` is passing. So i think it's not a real bc break. You could pass any `https` value in `$server => request()`, it always checks against the url scheme  via `parse_url`:

https://github.com/symfony/symfony/blob/081c60124318c8b6f9882a6bb96f656de8e00696/src/Symfony/Component/BrowserKit/Client.php#L305